include/coro/expected.hpp

```diff
@@ -11,7 +11,7 @@ template<typename E>
 using unexpected = std::unexpected<E>;
 } // namespace coro
 #else
-    #include <tl/expected.hpp>
+    #include "coro/detail/tl_expected.hpp"
```

Comment:
ok cool, this is a good solution, and works great moving forward with the std::expected change.

Can you update the readme-template to exclude this as a submodule dependency please?

Reply:
Will do